src/style.css

```diff
@@ -1,3 +1,90 @@
+html {
+  --active-link-color: #00acdc;
+  --menu-item-height: 60px;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.navigation {
+  margin-right: 50px;
+}
```

Comment:
It's better to define these spaces with paddings to `header`. I mean this one too
```css
.logo {
  margin-left: 50px;
}
```

Reply:
Not implemented too. Instead of making two of these margins you can add one `padding: 0 50px` to `.header`